standard.mk

```diff
@@ -41,33 +41,21 @@ default: gobuild
 
 .PHONY: clean
 clean:
-	rm -rf ./build/_output bundles-staging bundles-production
-	${CONTAINER_ENGINE} rmi \
-		${OPERATOR_IMAGE_URI} \
-		${OPERATOR_IMAGE_URI_LATEST} \
-		quay.io/${CATALOG_REGISTRY_ORGANIZATION}/$(OPERATOR_NAME):staging-latest \
-		quay.io/${CATALOG_REGISTRY_ORGANIZATION}/$(OPERATOR_NAME):production-latest 2>/dev/null || true
+	rm -rf ./build/_output
+
+.PHONY: isclean
+isclean:
+	@(test "$(ALLOW_DIRTY_CHECKOUT)" != "false" || test 0 -eq $$(git status --porcelain | wc -l)) || (echo "Local git checkout is not clean, commit changes and try again." >&2 && exit 1)
 
 .PHONY: build
-build: envtest
-	${CONTAINER_ENGINE} build . -f ${OPERATOR_DOCKERFILE} -t ${OPERATOR_IMAGE_URI}
-	${CONTAINER_ENGINE} tag ${OPERATOR_IMAGE_URI} ${OPERATOR_IMAGE_URI_LATEST}
+build: isclean envtest
+	$(CONTAINER_ENGINE) build . -f $(OPERATOR_DOCKERFILE) -t $(OPERATOR_IMAGE_URI)
+	$(CONTAINER_ENGINE) tag $(OPERATOR_IMAGE_URI) $(OPERATOR_IMAGE_URI_LATEST)
 
 .PHONY: push
 push:
-	${CONTAINER_ENGINE} push ${OPERATOR_IMAGE_URI}
-	${CONTAINER_ENGINE} push ${OPERATOR_IMAGE_URI_LATEST}
-
-.PHONY: skopeo-push
-skopeo-push: container-build
-	skopeo copy \
-		--dest-creds "${QUAY_USER}:${QUAY_TOKEN}" \
-		"docker-daemon:${OPERATOR_IMAGE_URI_LATEST}" \
-		"docker://${OPERATOR_IMAGE_URI_LATEST}"
-	skopeo copy \
-		--dest-creds "${QUAY_USER}:${QUAY_TOKEN}" \
-		"docker-daemon:${OPERATOR_IMAGE_URI}" \
-		"docker://${OPERATOR_IMAGE_URI}"
```

Comment:
This is referenced in build/deploy.sh

Reply:
You're right. Bleh. Yeah, we have separate build targets in different repos. Will add these back.